src/test/java/com/google/api/generator/engine/writer/JavaWriterVisitorTest.java

```diff
@@ -380,6 +381,72 @@ public void writeJavaDocComment_specialChar() {
     assertEquals(writerVisitor.write(), expected);
   }
 
+  @Test
+  public void writeMultiLineComment_apacheLicense() {
+    MultiLineComment apacheLicense = MultiLineComment.APACHE_LICENSE;
+    apacheLicense.accept(writerVisitor);
+    assertEquals(
+        writerVisitor.write(),
+        String.format(
+            createLines(15),
+            "/*\n",
+            "* Copyright 2020 Google LLC\n",
+            "*\n",
+            "* Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+            "* you may not use this file except in compliance with the License.\n",
+            "* You may obtain a copy of the License at\n",
+            "*\n",
+            "*     https://www.apache.org/licenses/LICENSE-2.0\n",
+            "*\n",
+            "* Unless required by applicable law or agreed to in writing, software\n",
+            "* distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+            "* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+            "* See the License for the specific language governing permissions and\n",
+            "* limitations under the License.\n",
+            "*/\n"));
+  }
+
+  @Test
+  public void writeMultiLineComment_sampleImports() {
+    MultiLineComment multiLineComment =
+        MultiLineComment.builder()
+            .addComment("Please include the following imports to run this sample.")
+            .addComment("import com.google.api.gax.rpc.ApiStreamObserver;")
+            .addComment("import com.google.example.library.v1.DiscussBookRequest;")
+            .addComment("import com.google.example.library.v1.LibraryClient;")
+            .build();
+    String expected =
+        String.format(
+            createLines(6),
+            "/*\n",
+            "* Please include the following imports to run this sample.\n",
+            "* import com.google.api.gax.rpc.ApiStreamObserver;\n",
+            "* import com.google.example.library.v1.DiscussBookRequest;\n",
+            "* import com.google.example.library.v1.LibraryClient;\n",
+            "*/\n");
+    multiLineComment.accept(writerVisitor);
+    assertEquals(writerVisitor.write(), expected);
+  }
+
+  @Test
+  public void writeMultiLineComment_specialChar() {
+    // TODO(xiaozhenliu): enable the check after html escaper is in place.
+    MultiLineComment multiLineComment =
+        MultiLineComment.builder()
+            .addComment("This may contain special characters like \t\b\r\n\f\"")
+            // .addComment("Also */ or <> & which should be handled by htmlEscaper.")
```

Comment:
I see this is a WIP PR, let's make a mental note of this to ensure we don't check in commented-out code.